Source/JavaScriptCore/runtime/ClonedArguments.cpp

```diff
@@ -49,7 +49,11 @@ ClonedArguments* ClonedArguments::createEmpty(VM& vm, Structure* structure, JSFu
 
     if (UNLIKELY(structure->mayInterceptIndexedAccesses() || structure->storedPrototypeObject()->needsSlowPutIndexing())) {
         if (!butterfly) {
-            butterfly = createArrayStorageButterfly(vm, nullptr, structure, length, vectorLength);
```

Comment:
For `ClonedArguments::createEmpty` signature, let's take `JSGlobalObject*` as a first parameter. In general, we take that when the function throws an error.

Reply:
Fixed.